gtrending/cli.py

```diff
@@ -0,0 +1,187 @@
+import argparse
+import json
+
+from typing import Dict, Any
+
+from .fetch import (
+    fetch_repos,
+    fetch_developers,
+    languages_list,
+    spoken_languages_list,
+)
+
+
+def main():
+
+    parser = argparse.ArgumentParser('Gtrending')
```

Comment:
This will be shown in the help text, let's change this to match how the program is run

```suggestion
    parser = argparse.ArgumentParser('gtrending', description='Query the GitHub Trending page')
```

Perhaps a description too?